core/indexer/errors.go

```diff
@@ -19,8 +19,8 @@ var ErrNoMiniblocks = errors.New("elasticsearch - no miniblocks")
 //ErrEmptyUserName signals that user name for elastic search is empty
 var ErrEmptyUserName = errors.New("user name is empty")
 
-//ErrEmptyPassword signals that password for elastic search is empty
-var ErrEmptyPassword = errors.New("password is empty")
+//ErrEmptyPassword signals that Password for elastic search is empty
```

Comment:
password * - that is why I hate refactoring process stuff :(

Reply:
yes, I know. Didn't pay attention to what it will rename. fixed